aidial_adapter_vertexai/chat/gemini/adapter.py

```diff
@@ -185,9 +185,13 @@ async def process_chunks(
             if chunk.candidates:
                 candidate = chunk.candidates[0]
 
-                content = candidate.text
-                await consumer.append_content(content)
-                yield content
+                try:
+                    content = candidate.text
+                except Exception:
```

Comment:
What's this for?

Reply:
To avoid an exception on empty content. Rewritten to make it more evident.